### ISSUE TITLE
Adding feature - verify/decode of QR encoded text file 

### ISSUE THREAD

**Issue by 3elwa47** [open-keychain/open-keychain#2739]
## Expected Behavior

It would be great if the app could read a QR encoded text file to decode the text and/or verify the signature. I think that would be a great feature to have.

Just a suggestion.

Thanks

**Comment by jlariviere1** [comment]
I agree; or if we could just paste text that includes a PGE Signed Message block
